sbncode/FlashMatch/FlashPredict_module.cc

```diff
@@ -142,14 +146,15 @@ FlashPredict::FlashPredict(fhicl::ParameterSet const& p)
 
   if (fMakeTree) initTree();
 
+  fAllPlanes = ((fSBND && fPlaneList==kSBNDPlanes) ||
```

Comment:
*I think* you could put these checks in the initialiser list so that `fAllPlanes` can be a `const`, which would be useful since you're comparing it often.